usaspending_api/settings.py

```diff
@@ -197,6 +201,7 @@
     "django.contrib.postgres",
     "django.contrib.staticfiles",
     # Third-party
+    "opentelemetry",
```

Comment:
Minor but please alphabetize this in the `third-party` section

Reply:
Done